### ISSUE TITLE
15 rules skipped due to selector errors:   .ui.top.attached.label+[class*="right floated"]+* -> Cannot read properties of undefined (reading 'type')

### ISSUE THREAD

**Issue by SimonSiefke** [okta/samples-js-angular#166]
![login-page](https://github.com/okta/samples-js-angular/assets/23744935/7b1e76e1-ea8c-4de5-bc0f-4d8327318436)

### Expected Behaviour
When running the custom login page example, no warning is shown in the console.

### Actual Behaviour

This warning is logged to the console:

```
polyfills.js:1 [webpack-dev-server] WARNING
15 rules skipped due to selector errors:
  .ui.top.attached.label+[class*="right floated"]+* -> Cannot read properties of undefined (reading 'type')
  .ui.top.attached.label+:not(.attached) -> Cannot read properties of undefined (reading 'type')
  .ui.modal>.icon+* -> Cannot read properties of undefined (reading 'type')
  #okta-sign-in .infobox h2+* -> Cannot read properties of undefined (reading 'type')
  #okta-sign-in .infobox h3+* -> Cannot read properties of undefined (reading 'type')
  #okta-sign-in .infobox li+* -> Cannot read properties of undefined (reading 'type')
  #okta-sign-in .infobox ol+* -> Cannot read properties of undefined (reading 'type')
  #okta-sign-in .infobox p+* -> Cannot read properties of undefined (reading 'type')
  #okta-sign-in .infobox ul+* -> Cannot read properties of undefined (reading 'type')
  #okta-sign-in .infobox-compact h2+* -> Cannot read properties of undefined (reading 'type')
  #okta-sign-in .infobox-compact h3+* -> Cannot read properties of undefined (reading 'type')
  #okta-sign-in .infobox-compact li+* -> Cannot read properties of undefined (reading 'type')
  #okta-sign-in .infobox-compact ol+* -> Cannot read properties of undefined (reading 'type')
  #okta-sign-in .infobox-compact p+* -> Cannot read properties of undefined (reading 'type')
  #okta-sign-in .infobox-compact ul+* -> Cannot read properties of undefined (reading 'type')
```

**Comment by shuowu-okta** [comment]
Thanks for reporting the issue!

Internal Ref: OKTA-627424

**Comment by jaredperreault-okta** [comment]
Resolved in #168
